extendr-api/src/robj/try_from_robj.rs

```diff
@@ -537,45 +527,112 @@ macro_rules! impl_try_from_robj {
         }
     };
 }
-
 #[rustfmt::skip]
-impl_try_from_robj!(
-    u8, u16, u32, u64, usize,
-    i8, i16, i32, i64, isize,
-    bool,
-    Rint, Rfloat, Rbool, Rcplx,
-    f32, f64,
-    Vec::<String>,
-    HashMap::<String, Robj>, HashMap::<&str, Robj>,
-    Vec::<Rint>, Vec::<Rfloat>, Vec::<Rbool>, Vec::<Rcplx>, Vec::<u8>, Vec::<i32>, Vec::<f64>,
-    &[Rint], &[Rfloat], &[Rbool], &[Rcplx], &[u8], &[i32], &[f64],
-    &mut [Rint], &mut [Rfloat], &mut [Rbool], &mut [Rcplx], &mut [u8], &mut [i32], &mut [f64],
-    &str, String,
-);
+impl_try_from_robj!(u8);
+impl_try_from_robj!(u16);
+impl_try_from_robj!(u32);
+impl_try_from_robj!(u64);
+impl_try_from_robj!(usize);
+
+impl_try_from_robj!(i8);
+impl_try_from_robj!(i16);
+impl_try_from_robj!(i32);
+impl_try_from_robj!(i64);
+impl_try_from_robj!(isize);
+
+impl_try_from_robj!(bool);
+
+impl_try_from_robj!(Rint);
+impl_try_from_robj!(Rfloat);
+impl_try_from_robj!(Rbool);
+impl_try_from_robj!(Rcplx);
+
+impl_try_from_robj!(f32);
+impl_try_from_robj!(f64);
+
+impl_try_from_robj!(Vec::<String>);
+impl_try_from_robj!(Vec::<Rint>);
+impl_try_from_robj!(Vec::<Rfloat>);
+impl_try_from_robj!(Vec::<Rbool>);
+impl_try_from_robj!(Vec::<Rcplx>);
+impl_try_from_robj!(Vec::<u8>);
+impl_try_from_robj!(Vec::<i32>);
+impl_try_from_robj!(Vec::<f64>);
+
+impl_try_from_robj!(&[Rint]);
+impl_try_from_robj!(&[Rfloat]);
+impl_try_from_robj!(&[Rbool]);
+impl_try_from_robj!(&[Rcplx]);
+impl_try_from_robj!(&[u8]);
+impl_try_from_robj!(&[i32]);
+impl_try_from_robj!(&[f64]);
+
+impl_try_from_robj!(&mut [Rint]);
+impl_try_from_robj!(&mut [Rfloat]);
+impl_try_from_robj!(&mut [Rbool]);
+impl_try_from_robj!(&mut [Rcplx]);
+impl_try_from_robj!(&mut [u8]);
+impl_try_from_robj!(&mut [i32]);
+impl_try_from_robj!(&mut [f64]);
+
+impl_try_from_robj!(&str);
+impl_try_from_robj!(String);
```

Comment:
@CGMossa This appears to be a formatting change of the previous single macro invocation. is that correct?

Reply:
Yes. Instead of providing all the types at once, it was now necessary to call the macro on each type separately.